src/zenml/cli/example.py

```diff
@@ -176,9 +294,45 @@ def examples(self) -> List[Example]:
                 not name.startswith(".")
                 and not name.startswith("__")
                 and not name.startswith("README")
+                and not name.endswith(".sh")
             )
         ]
 
+    @property
+    def is_installed(self) -> bool:
+        """Checks if the"""
```

Comment:
Missing docstring :-)

Reply:
Ah I missed this one, the method is no longer used. Will remove it altogether